examples/src/main/java/alluxio/examples/UnderFileSystemCommonOperations.java

```diff
@@ -237,15 +237,23 @@ public void createOpenExistingLargeFileTest() throws IOException {
     int numCopies = prepareMultiBlockFile(testFile);
     InputStream inputStream = mUfs.openExistingFile(testFile);
     byte[] buf = new byte[numCopies * TEST_BYTES.length];
-    int bytesRead = inputStream.read(buf, 0, buf.length);
-    if (buf.length != bytesRead) {
-      throw new IOException(FILE_CONTENT_INCORRECT);
-    }
-    for (int i = 0; i < bytesRead; ++i) {
-      if (TEST_BYTES[i % TEST_BYTES.length] != buf[i]) {
-        throw new IOException(FILE_CONTENT_INCORRECT);
+    int offset = 0;
+    while (offset < buf.length) {
+      int bytesRead = inputStream.read(buf, offset, buf.length - offset);
+      if (bytesRead != -1) {
```

Comment:
nit: break first and remove indent for the if block

Reply:
Changed